server/src/seeders/20201217175111-demo-cutomers-orders.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+const faker = require('faker');
+const v4 = require("uuid").v4;
+const bcrypt =  require('bcryptjs');
+
+const SALT_FACTOR = 10;
+        
+function getRandomInt(max) {
+    return Math.floor(Math.random() * Math.floor(max));
+};
+
+function randomTime(start, end) {
+    var diff =  end.getTime() - start.getTime();
+    var new_diff = diff * Math.random();
+    var date = new Date(start.getTime() + new_diff);
+    return date;
+}
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        
+        //200 User 
+        let users = [];
+        for (let i = 0; i <= 199; i++) {
+            const hashedPassword = await bcrypt.hash(faker.internet.password(), SALT_FACTOR);
+            const u = randomTime(new Date("01-01-2020 10:30"), new Date("12-31-2020 02:10"));
```

Comment:
Bitte Relativ machen - Bsp heutiges Datum, vor einem Jahr & heutiges Datum
-> So wird vermieden, dass du z.B. Sachen in der Zukunft anlegst